README.md

```diff
@@ -82,6 +82,10 @@ On bicycle it could take less, it is convenient  and extremely popular mean of t
 
 **Mikhail Davydov**: Social security, stable economy, it cares of environment, easy to get to any point of europe. 
 
+### How big are the taxes for foreigners?
```

Comment:
very important advice for first year esp.: you can return a lot taxes because you earn less in your home land, or if you paid taxes in homeland. Ask your steuerberater for this issue.